readme-vars.yml

```diff
@@ -29,19 +29,32 @@ param_ports:
 # application setup block
 app_setup_block_enabled: true
 app_setup_block: "
-* To log in to the application, browse to https://<hostip>:9000.
+* When the application first runs, it will populate its /config
+  * (be sure you have user permissions matched between the host and the container by using PUID and PGID)
```

Comment:
This line isn't needed. PUID and PGID are required for all of our containers, and documented already.